doc/recipe/preprocessor.rst

```diff
@@ -247,6 +247,13 @@ available tables of the specified project.
    a given dataset) fx files are found in more than one table, ``mip`` needs to
    be specified, otherwise an error is raised.
 
+.. note::
+   To explicitly **not** use any fx variables in a preprocessor, use either
+   ``fx_variables: null``, ``fx_variables: {}`` or ``fx_variables: []``.  While
```

Comment:
I find it confusing to have multiple options for the same functionality - decide which one single one you like and use only that, you can be fancy and add a catchment in the recipe parser eg `if fx_variables == {} or fx_variables == []: raise RecipeError("I see you are trying to be smart, use null for fx_variables, smartypants")`

Reply:
Right now we allow `dicts` and `lists` for fx variables (see [here](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html#fx-variables-as-cell-measures-or-ancillary-variables)), so I think at least an empty `list` and an empty `dict` should be allowed. For me it feels natural to also inlcude `fx_variables=None`, but I can remove that if necessary :smile:

Reply:
I am more in favour of `Null` or `None` only - think from a user's perspective and not from the developer's perspective - how nice is it when you read the Numpy docs and they simply say `set this to Null if you don't want to have it` rather than `you can call your grandfather Grampa, Gramps, as well as Granddad, as well as Pops etc`... UX first, ma man

Reply:
How about we allow all of them but just mention `fx_variables: null` in the doc?

Reply:
sounds very good to me :beer: